cmd/podman/pod_create.go

```diff
@@ -71,6 +94,9 @@ func podCreateCmd(c *cli.Context) error {
 			return errors.Wrapf(err, "unable to write pod id file %s", c.String("pod-id-file"))
 		}
 	}
+	if !c.BoolT("pause") && c.IsSet("share") {
```

Comment:
Should also check that share isn't set to "" - sharing nothing with no pause container is acceptable.